templates/corporate/billing.html

```diff
@@ -141,6 +141,9 @@ <h3>
                 </div>
                 {% elif admin_access and not has_active_plan %}
                 <div class="tab-content">
+                    {% if sponsorship_pending %}
+                    <h3>Your organization's sponsorship request for discount on Zulip Standard is being processed. We will get back to you soon.</h3>
```

Comment:
"Your organization's request for sponsorship is being processed ..." might be cleaner?

Reply:
hmm. This works. I wanted to include discount in the sentence because otherwise, someone else who opens /billing might think the realm is going be fully sponsored.